vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -83,6 +83,10 @@
      */
     public DatePicker() {
```

Comment:
Seeing how the constructor in `GeneratedDatePicker` only calls `setPresentationValue` if the initial value is not null, there are several constructor variants where `setPresentationValue` still might not be called:

For example the following test fails:
```java
    @Test
    public void initialNullValue_valuePropertyHasEmptyString() {
        // Assume we load a nullable LocalDate value from somewhere and initialize the date picker with it
        LocalDate initialDateFromDatabase = null;
        DatePicker picker = new DatePicker((LocalDate) null);
        Assert.assertNull(picker.getValue());
        Assert.assertEquals("", picker.getElement().getProperty("value"));
    }
```

Reply:
Good catch, updated the PR.